pkg/cluster/cluster.go

```diff
@@ -148,6 +154,7 @@ func (c *Cluster) setup() error {
 }
 
 func (c *Cluster) create() error {
+	c.logger.Infof("creating cluster (%s): spec: %#v, status: %#v", c.cluster.Name, c.cluster.Spec, c.cluster.Status)
```

Comment:
creating cluster with Spec, Status. Removing the name part. it is already in the logger.